### DISCUSSION TITLE
Resolving the Serenity.CoreLib.js and React Dependency Error When Using Preact

### DISCUSSION THREAD

**Discussion by s10730047** [serenity-is/Serenity#7152] [Help]
There is an error: "To use React, it should be included before Serenity.CoreLib.js." When using CardViewMixin, I did not install React in the project, so React isn't included after appsettings.bundles.json. I'm using Preact. Can you help me adjust the grammar?

 My project Serenity version is 8.4.4
`
  new CardViewMixin({
      grid: this as DataGrid<any, any>,
      renderItem: (item) => h(CustomerCard, {
          item: item,
          editItem: editItem
      })
  });
`

**Comment by VictorTomaili** [comment]
CardViewMixin is a pro feature, please ask the question to the StartSharp repository or email.
